tests/lib/rules/require-return-from-computed.js

```diff
@@ -102,5 +87,26 @@ eslintTester.run('require-return-from-computed', rule, {
         },
       ],
     },
+    {
+      // Decorator (with parenthesis):
+      code: 'class Test { @computed() get someProp() {} }',
```

Comment:
Since these examples should be using the import to determine if this is the right “computed” (I know we haven’t fully implemented that) could we add the import to the code? That way, when we _actually_ update to use the imports properly we have “one less test to fix”?